airgun/browser.py

```diff
@@ -178,6 +179,8 @@ def _get_selenium_browser(self):
             prefs = {'download.prompt_for_download': False}
             options.add_experimental_option("prefs", prefs)
             options.add_argument('disable-web-security')
+            if browseroptions:
+                options.add_argument(browseroptions)
```

Comment:
Can you make that:
```python
if browseroptions:
    for opt in browseroptions.split(';'):
        options.add_argument(opt)
```
?

If we name setting "browseroptions", we really should support multiple options ;) .

We will probably need to eventually change this somehow, but using semicolon as delimiter sound like relatively safe idea. I know for sure that some options take argument of comma-delimited list, so we want to use something other than comma.

Reply:
@mirzal done